nova.mix.js

```diff
@@ -27,7 +27,7 @@ class NovaExtension {
       ...(webpackConfig.resolve.alias || {}),
       'laravel-nova': path.join(
         __dirname,
-        'vendor/laravel/nova/resources/js/mixins/packages.js'
+        '../../vendor/laravel/nova/resources/js/mixins/packages.js'
```

Comment:
```suggestion
        'vendor/laravel/nova/resources/js/mixins/packages.js'
```

This reverts another change that was made

> The path was updated from '../../vendor' to ensure the package functions correctly as a standalone package. This change aligns with the [Laravel Nova documentation on mixins (part 2)](https://nova.laravel.com/docs/customization/frontend#using-nova-mixins), which states that standalone packages should not depend on the main project structure.

Reply:
Hi,

Unfortunatly, I've got the issue with all my packages, and I don't know how to resolve it ... I work in nova-components directory but when I try to build package it never found laravel-nova  alias  so I must change the path.

If you give me a hand with this, I fix it quickly
Regards